lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -2462,6 +2483,93 @@ class CommandObjectTargetModulesDumpLineTable
   CommandOptions m_options;
 };
 
+#pragma mark CommandObjectTargetModulesDumpSeparateDebugInfoFiles
+
+// Image debug dwo dumping command
+
+class CommandObjectTargetModulesDumpSeparateDebugInfoFiles
+    : public CommandObjectTargetModulesModuleAutoComplete {
+public:
+  CommandObjectTargetModulesDumpSeparateDebugInfoFiles(
+      CommandInterpreter &interpreter)
+      : CommandObjectTargetModulesModuleAutoComplete(
+            interpreter, "target modules dump separate-debug-info",
+            "Dump the separate debug info symbol files for one or more target "
+            "modules.",
+            //"target modules dump separate-debug-info [<file1> ...]")
```

Comment:
This is commented out why?

Reply:
I was following the pattern I noticed for some of the other classes like `CommandObjectTargetModulesDumpSections` where they add the help message as a comment. I don't mind removing it though, since it doesn't seem like *all* of the classes do it.